loop/models.py

```diff
@@ -56,6 +58,8 @@ class State(LoopModel):
     state_name_en = models.CharField(max_length=100, null=True)
     helpline_number = models.CharField(max_length=14, null=False, blank=False, default="0")
     crop_add = models.BooleanField(default=False)
+    phone_digit = models.CharField(default=10, max_length=2, blank=True, null=True)
+    phone_start = models.CharField(default=789, max_length=15, blank=True, null=True)
```

Comment:
try to add a string field containing regex instead of using digits and start. Also this needs to be placed in the country model.